lib/iris/tests/unit/fileformats/netcdf/load_cube/load_cube__activate/test__grid_mappings.py

```diff
@@ -846,9 +846,9 @@ def test_aux_lat_rotated(self):
         # 	005 : fc_build_coordinate_latitude_rotated
```

Comment:
Shouldn't this change also affect the rules triggered?

Reply:
Yes, you're right -- I haven't updated these comments.
But they are only a reference for debug comparison.
I don't think it's worth fixing, as we are going to remove them all when we are done.

Reply:
Fair enough.